output/adc2vtk.pl

```diff
@@ -460,7 +461,7 @@
 printf VTKFLUXBOUNDARY "<VTKFile type=\"PolyData\" version=\"0.1\" byte_order=\"LittleEndian\">\n";
 printf VTKFLUXBOUNDARY "   <PolyData>\n";
 #
-# write out the flux-specified boundary tables as geometry
+# write out the flux-specified boundary tables as XDMF 3DSMESH geometry
 # to show boundary height
 my $xdmfFluxBoundaryGeometryFileName = $meshfile . "_fluxBoundaryGeometry.xmf";
 unless (open(XDMFFLUXBOUNDARY,">$xdmfFluxBoundaryGeometryFileName")) {
```

Comment:
existing `unless ...` is fine, I just recommend we not add more (don't change existing `unless` based on my comments above)